src/mdb_v8.c

```diff
@@ -2196,7 +2272,8 @@ jsobj_maybe_garbage(uintptr_t addr)
 	    type != V8_TYPE_JSARRAY &&
 	    type != V8_TYPE_JSFUNCTION &&
 	    type != V8_TYPE_JSDATE &&
-	    type != V8_TYPE_JSREGEXP)));
+	    type != V8_TYPE_JSREGEXP &&
+	    type != V8_TYPE_JSTYPEDARRAY)));
```

Comment:
Do you know if the changing representation of Buffer has affected the "::nodebuffer" dcmd?